### PR TITLE
MKM add 1 card

### PR DESCRIPTION
**Pull request by Wild-W**
Festerleech - spoiled 1/20/2024

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/festerleech.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Festerleech
+ManaCost:B
+Types:Creature Zombie Leech
+PT:1/1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigMill | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, you mill two cards.
+A:AB$ Pump | Cost$ 1 B | NumAtt$ +2 | NumDef$ +2 | ActivationLimit$ 1 | SpellDescription$ Festerleech gets +2/+2 until end of turn. 
+SVar:TrigMill:DB$ Mill | NumCards$ 2 | Defined$ You
```
Comment (tehdiplomat) [inline forge-gui/res/cardsfolder/upcoming/festerleech.txt:7]: I would recommend grouping this above the ability so its right next to the trigger that cares about it.

File: forge-gui/res/cardsfolder/upcoming/festerleech.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Festerleech
+ManaCost:B
+Types:Creature Zombie Leech
+PT:1/1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigMill | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, you mill two cards.
+A:AB$ Pump | Cost$ 1 B | NumAtt$ +2 | NumDef$ +2 | ActivationLimit$ 1 | SpellDescription$ Festerleech gets +2/+2 until end of turn.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/festerleech.txt:6]: replace `Festerleech` -> `CARDNAME`

File: forge-gui/res/cardsfolder/upcoming/festerleech.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Festerleech
+ManaCost:B
+Types:Creature Zombie Leech
+PT:1/1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigMill | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, you mill two cards.
+A:AB$ Pump | Cost$ 1 B | NumAtt$ +2 | NumDef$ +2 | ActivationLimit$ 1 | SpellDescription$ Festerleech gets +2/+2 until end of turn. 
+SVar:TrigMill:DB$ Mill | NumCards$ 2 | Defined$ You
+DeckHas:Ability$Graveyard
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/festerleech.txt:8]: Ability$Mill not Graveyard

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/festerleech.txt:8]: You could do DeckHints:Ability$Graveyard but this card doesn't actually make use of the graveyard other than to fill it

File: forge-gui/res/cardsfolder/upcoming/festerleech.txt (hunk 2 above)
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/festerleech.txt:6]: Add the activation limit to the `SpellDescription` or it won't show in card detail

Card detail always needs to match Oracle

File: forge-gui/res/cardsfolder/upcoming/festerleech.txt (hunk 3 above)
Comment (Wild-W) [reply forge-gui/res/cardsfolder/upcoming/festerleech.txt]: Resolved, though if this is the case, maybe Eye Collector needs to be looked at again, it uses `Ability$Graveyard`.

**Review by tool4ever — APPROVED** [review]

**Comment by Hanmac** [comment]
@tool4ever can we update the line ending please ?

**Comment by tool4ever** [comment]
to what end? it seems to be CRLF and loads fine

**Comment by Hanmac** [comment]
@tool4ever imo CRLF looks ugly to me and i hate it when the files have mixed Line Endings

**Comment by tool4ever** [comment]
hmn I really thought that's just the standard
or my Notepad is just lying

**Pull request merged**